cmd/generate-config/config/config-openapi-spec.json

```diff
@@ -69,6 +70,26 @@
         }
       }
     },
+    "ingress": {
```

Comment:
In your EP, you said you intended to expose this via:

```yaml
router:
  routerAdmissionPolicy:
    namespaceOwnership: <Strict|InterNamespaceAllowed> # Defaults to InterNamespaceAllowed.
```

here, you're exposing it via ingress.routerAdmissionPolicy.namespaceOwnership - if the implementation is what we want, you should probably go back and fixup the EP.

Reply:
Good catch, will update it right away.